src/lib/serviceRoutines/postSubscribeContext.cpp

```diff
@@ -36,6 +36,15 @@
 /* ****************************************************************************
 *
 * postSubscribeContext - 
+*
+* POST /v1/subscribeContext
+* POST /ngsi10/subscribeContext
+*
+* Payload In:  SubscribeContextRequest
+* Payload Out: SubscribeContextResponse
+*
+* URI parameters
+*   - notifyFormat=XXX    (handled by mongoBackend)
```

Comment:
Just to clarify, in the other service routine (postSubscribeContextConvOp) we noi longer treat 'entity::type=XXX', but we do treat 'exist=entity::type' and '!exist=entity::type'.
And I just realized I have no func-test for that :-( :-D